### PR TITLE
LOG_LEVELS should be defined in config.rb

### LINKED ISSUE #262: Exception thrown when log level is set by environment variable
```bash
export RUBY_DEBUG_LOG_LEVEL=INFO
./bin/rails console
```

results in:

```bash
11: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug.rb:3:in `<top (required)>'
10: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug.rb:3:in `require_relative'
 9: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/session.rb:6:in `<top (required)>'
 8: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/session.rb:6:in `require_relative'
 7: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/config.rb:3:in `<top (required)>'
 6: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/config.rb:335:in `<module:DEBUGGER__>'
 5: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/config.rb:335:in `new'
 4: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/config.rb:48:in `initialize'
 3: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/config.rb:196:in `parse_argv'
 2: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/config.rb:196:in `each'
 1: from /Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/config.rb:198:in `block in parse_argv'
/Users/dhh/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/debug-1.0.0/lib/debug/config.rb:174:in `parse_config_value': uninitialized constant DEBUGGER__::LOG_LEVELS (NameError)
```

Because debug.rb requires session.rb which requires config.rb at the top, before it's defined itself, which means that `DEBUGGER__::LOG_LEVELS` is not yet declared by the time config.rb wants to use it in `#parse_config_value`.

### PR DESCRIPTION
**Pull request by st0012**
Fixes #262

### CODE REVIEW

File: test/debug/config_test.rb [hunk 1]
```diff
@@ -303,4 +303,31 @@ def test_p_with_keep_alloc_site
...
+    def test_debugger_takes_log_levels_config
+      # default WARN level doesn't report threads creation
+      debug_code(program, remote: false) do
+        type 'c'
+        assert_no_line_text(/Thread #\d+ is created/)
+        type 'c'
+      end
+
+      ENV["RUBY_DEBUG_LOG_LEVEL"] = "INFO"
+      debug_code(program, remote: false) do
```
Comment (st0012) [inline test/debug/config_test.rb:324]: Because the current logging logic only uses standard output/error, it's complicated to test in remote modes.

**Comment by dhh** [comment]
@ko1 Would it be possible to get a 1.0.1 release that includes this fix? Then we can use that for Rails 7 alpha 1, and avoid having the debug message clutter all Rails invocations 🙏

**Comment by ko1** [comment]
@dhh Sorry for late response of your first issue.

(1) initial message

It's okay to remove the starting message (use `info` level).
It is not so important.

```
@@ -1519,7 +1535,7 @@ module DEBUGGER__

   class << self
     define_method :initialize_session do |ui|
-      DEBUGGER__.warn "Session start (pid: #{Process.pid})"
+      DEBUGGER__.info "Session start (pid: #{Process.pid})"

       ::DEBUGGER__.const_set(:SESSION, Session.new(ui))
```

BTW, I don't remove the initial message on remote debugging because it should be explicitly logged to avoid danger of opening debugging port unexpectedly.

```
[master]$ rdbg -O target.rb
DEBUGGER: Debugger can attach via UNIX domain socket (/home/ko1/.ruby-debug-sock/ruby-debug-ko1-18267)
```

(2) 1.0.1 release

No problem to release 1.0.1.
Do you have rails schedule?

We have other issues and I want to fix them as possible.

**Comment by dhh** [comment]
No problem. It would be much nicer if that session start message is sent on info, and the default log level is below that 🙏. Agree re: warning on the open port.

May release alpha 1 as soon as tomorrow. But we don't need to coordinate that close. It's OK if there's just a little noise on alpha 1. We have ">= 1.0.0" set for the debug gem, so as soon as you release 1.0.1, it'll automatically get picked up.

Thanks again for this awesome step up in Ruby debugging ❤️

**Pull request merged**